server.js

```diff
@@ -24,10 +25,19 @@ app.use(
 app.use(compression());
 
 const gitClone = (repoUrl) => {
+  const parsedUrl = new URL(repoUrl);
+
+  const userInfo =
```

Comment:
Let's remove userinfo since even username could be considered sensitive. so just the protocol, hostname and path.

Reply:
Good feedback, done